lib/odbc/test/odbc_test.hrl

```diff
@@ -30,9 +30,9 @@
 		   {unix,linux} ->
 		       case  erlang:system_info({wordsize, external}) of
 			   4 ->
-			       mysql;
+			       sqlserver;
 			   _ ->
-			       postgres
```

Comment:
This i not going to work out for the nightly builds! Also it eliminates all Postges test which probably is not a good thing.